coroutine/arm64/Context.h

```diff
@@ -66,12 +78,17 @@ static inline void coroutine_initialize(
 
     // Stack grows down. Force 16-byte alignment.
     char * top = (char*)stack + size;
-    context->stack_pointer = (void**)((uintptr_t)top & ~0xF);
+    top = (char *)((uintptr_t)top & ~0xF);
+    context->stack_pointer = (void**)top;
 
     context->stack_pointer -= COROUTINE_REGISTERS;
     memset(context->stack_pointer, 0, sizeof(void*) * COROUTINE_REGISTERS);
 
+#if defined(__ARM_FEATURE_PAC_DEFAULT) && __ARM_FEATURE_PAC_DEFAULT != 0
```

Comment:
What do you think about avoiding `#if` here and making `ptrauth_sign_instruction_addr` a no-op on platforms where it's not enabled? That way we can avoid increasing cyclomatic complexity.

Reply:
Sounds reasonable to me 👍